commit 0922360ec29816a0843f5eb93be24891ef2cfe8f
Author: Volker Krause <vkrause@kde.org>
Date:   Fri Aug 4 20:15:45 2017 +0200

    Don't use find_file for finding files in our own source dir
    
    Summary:
    It's not necessary, and it breaks when cross-compiling, as find_file
    changes its behavior then.
    
    Reviewers: #frameworks
    
    Tags: #frameworks
    
    Differential Revision: https://phabricator.kde.org/D7136

diff --git a/src/runtime/kwalletd/CMakeLists.txt b/src/runtime/kwalletd/CMakeLists.txt
index f530d7a..7c916c4 100644
--- a/src/runtime/kwalletd/CMakeLists.txt
+++ b/src/runtime/kwalletd/CMakeLists.txt
@@ -78,7 +78,7 @@ if (Gpgmepp_FOUND)
     )
 endif(Gpgmepp_FOUND)
 
-find_file(kwallet_xml org.kde.KWallet.xml HINTS ${CMAKE_CURRENT_SOURCE_DIR}/../../api/KWallet)
+set(kwallet_xml ${CMAKE_CURRENT_SOURCE_DIR}/../../api/KWallet/org.kde.KWallet.xml)
 
 qt5_add_dbus_adaptor( kwalletd_SRCS ${kwallet_xml} kwalletd.h KWalletD )
 
